notebooks/vertex_genai/solutions/deployment_adk_agent.ipynb

```diff
@@ -0,0 +1,1061 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    agent_engines.update(
</pre><p> </p><p>It need to be:</p>

<pre class="ql-syntax" spellcheck="false">remote_app = agent_engines.update(...
</pre><p>to call the updated app in the next cell. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>